### PR TITLE
AppVeyor API URL

### PR DESCRIPTION
**Pull request by ocefpaf**
Hopefully this will solve https://github.com/conda-forge/staged-recipes/issues/221

Ping @jakirkham

### CODE REVIEW

File: conda_smithy/ci_register.py [hunk 1]
```diff
@@ -108,6 +108,8 @@ def appveyor_encrypt_binstar_token(feedstock_directory, user, project):
 def appveyor_configure(user, project):
     """Configure appveyor so that it skips building if there is no appveyor.yml present."""
     headers = {'Authorization': 'Bearer {}'.format(appveyor_token)}
+    # I have reasons to believe this is all AppVeyor is doing to the API URL.
+    project = project.replace('_', '-')
```
Comment (jakirkham) [inline conda_smithy/ci_register.py]: Sounds believable from what I have seen too.

**Comment by jakirkham** [comment]
LGTM

**Comment by jakirkham** [comment]
Any chance we could get this as `v0.7.2`?

**Comment by pelson** [comment]
Done. Thanks for nudging @jakirkham

**Comment by jakirkham** [comment]
Awesome! Thanks, @pelson.

**Comment by jakirkham** [comment]
Fixes https://github.com/conda-forge/staged-recipes/issues/221

**Pull request merged**